test/helpers/last_menstrual_period_helper_test.rb

```diff
@@ -3,94 +3,101 @@
 class LastMenstrualPeriodHelperTest < ActionView::TestCase
   before do
     @user = create :user
-    @patient = create :patient, created_by: @user,
-                                initial_call_date: 2.days.ago,
-                                appointment_date: 2.days.from_now
-
     @pregnancy = create :pregnancy,
                         last_menstrual_period_weeks: 9,
                         last_menstrual_period_days: 2,
                         created_by: @user
+    @patient = create :patient, created_by: @user,
+                                initial_call_date: 2.days.ago,
+                                appointment_date: 2.days.from_now,
+                                pregnancy: @pregnancy
```

Comment:
this is what i had to do to fix the existing tests, patient needs to include pregnancy so the patient info can be pulled into the tests